README.md

```diff
@@ -71,17 +104,19 @@ Work is planned to have a new method to handle CSR for external signed CAs in a
 Usage
 =====
 
+
 How to use ansible-freeipa
 --------------------------
 
 **GIT repo**
 
 The simplest method for now is to clone this repository on the controller from github directly and to start the deployment from the ansible-freeipa directory:
 
-```bash
+```
```

Comment:
Why do you remove `bash` and `yaml`?

Reply:
I removed them for better rendering on tools that do render Markdown.

"bash" was used for shell commands, and not bash scripts, and the "yaml" removed were used on INI files (invertory), and not on YAML files. Unless I'm missing something, I don't think they are needed.